Integrations/AbuseDB/AbuseDB.py

```diff
@@ -208,14 +208,19 @@ def createEntry(context_ip, context_ip_generic, human_readable, dbot_scores, tit
 
 
 def check_ip_command(ip, days=MAX_AGE, verbose=VERBOSE, threshold=THRESHOLD):
-    params = {
-        "ipAddress": ip,
-        "maxAgeInDays": days
-    }
-    if verbose:
-        params['verbose'] = "verbose"
-    analysis = http_request("GET", url_suffix=CHECK_CMD, params=params).get("data")
-    return analysis_to_entry(analysis, verbose=verbose, threshold=threshold)
+
+    ip_list = argToList(ip)
+    entry_list = []
+    for corrent_ip in ip_list:
+        params = {
```

Comment:
lets initialize `params` out of the loop, and then assign the ip address to it here